bouncer/src/repo/manipulator/modelconvertor/export/repo_model_export_src.cpp

```diff
@@ -103,6 +107,8 @@ SRCModelExport::SRCModelExport(
 	const repo::core::model::RepoScene *scene
 	) : WebModelExport(scene)
 {
+	enableCompression = true;
```

Comment:
use REPO_BOOST_NO_GZIP, like in `repo_model_import_3drepo.cpp`

I think for our current use case we'd always want to gzip this. the only situation is if zlib isn't compiled into boost (default windows boost doesn't come with it, I had to manually compile it  :( ).

the REPO_BOOST_NO_GZIP flag is used so we can still compile 3drepobouncer if it doesn't exist, which is probably more appropriate than a hard coded flag

Reply:
@carmenfan that does make more sense. Fixed.